wgpu/Cargo.toml

```diff
@@ -73,6 +73,9 @@ glsl = ["naga/glsl-in", "wgc/glsl"]
 ## Enable accepting WGSL shaders as input.
 wgsl = ["wgc?/wgsl"]
 
+## Enables the static DXC compiler using the `mach-dxcompiler-rs` crate.
+mach-dxcompiler-rs = ["hal/mach-dxcompiler-rs"]
+
```

Comment:
Could we move this to its own section for "external libraries" and explain a little more why we might want to use this? I should be able to fully answer why I might care by looking at this.

Reply:
Similarly here, I would recommend naming the feature `static-dxc`.